Orochi/OrochiUtils.h

```diff
@@ -18,10 +18,13 @@ class OrochiUtils
 		int x, y, z, w;
 	};
 
-	static oroFunction getFunctionFromFile( oroDevice device, const char* path, const char* funcName, std::vector<const char*>* opts );
-	static oroFunction getFunctionFromString( oroDevice device, const char* source, const char* path, const char* funcName, std::vector<const char*>* opts, 
+	OrochiUtils();
+	~OrochiUtils();
```

Comment:
```suggestion

```

Reason: we can default initialize directly on the member variables `m_cacheDirectory` and do not need the explicitly defined constructor.